spec/controllers/locations_controller_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe LocationsController, type: :controller do
+  describe "#external_search_action" do
+    # Extracted from LocationsController#external_search.
+    # See tests for external_search path in test/controllers/locations_controller_test.rb.
```

Comment:
Any plans to migrate these?

Reply:
moved two to get the ball rolling 👍 will move the others in the file gradually